alibi/prototypes/protoselect.py

```diff
@@ -568,32 +564,31 @@ def visualize_image_prototypes(summary: 'Explanation',
         :py:meth:`alibi.prototypes.protoselect.ProtoSelect.summarise` method.
     trainset
         Tuple, `(X_train, y_train)`, consisting of the training data instances with the corresponding labels.
-    reducer
-        2D reducer. Reduces the input feature representation to 2D. Note that the reducer operates directly on the
-        input instances if ``preprocess_fn=None``. If the `preprocess_fn` is specified, the reducer will be called
-        on the feature representation obtained after passing the input instances through the `preprocess_fn`.
     preprocess_fn
-        Preprocessor function.
+        Optional preprocessor function. If ``preprocess_fn=None``, no preprocessing is applied.
     knn_kw
         Keyword arguments passed to `sklearn.neighbors.KNeighborsClassifier`. The `n_neighbors` will be
         set automatically to 1, but the `metric` has to be specified according to the kernel distance used.
         If the `metric` is not specified, it will be set by default to ``'euclidean'``.
         See parameters description:
         https://scikit-learn.org/stable/modules/generated/sklearn.neighbors.KNeighborsClassifier.html
-    ax
-        A `matplotlib` axes object to plot on.
-    fig_kw
-        Keyword arguments passed to the `fig.set` function.
-    image_size
-        Shape to which the prototype images will be resized. A zoom of 1 will display the image having the shape
-        `image_size`.
-    zoom_lb
-        Zoom lower bound. The zoom will be scaled linearly between `[zoom_lb, zoom_ub]`.
-    zoom_ub
-        Zoom upper bound. The zoom will be scaled linearly between `[zoom_lb, zoom_ub]`.
+
+    Returns
+    -------
+    A dictionary containing:
+
+     - ``'prototypes_indices'`` - an array of the prototypes indices.
+
+     - ``'prototypes_importance'`` - an array of prototypes importance.
```

Comment:
I think more grammatically correct would be to call these `prototype_indices` and `prototype_importances`?

Reply:
Similar for the name of the function.